app/src/main/res/values/strings.xml

```diff
@@ -186,6 +186,7 @@
   </plurals>
   <string name="bar_separator">\u0020|\u0020</string>
   <!-- ProfileChooserFragment -->
+  <string name="profile_chooser_label">Profile selection page</string>
```

Comment:
Perhaps instead: ``profile_chooser_activity_label`` to clarify this specifically pertains to an activity.

Reply:
Done. Resolving this now.